sample-apps/http-server/Dockerfile

```diff
@@ -5,6 +5,7 @@ COPY . .
 ENV GOPROXY direct
 
 RUN go mod download
+RUN go mod download github.com/aws/aws-lambda-go
```

Comment:
Hmm this seems strange, why don't we have to do this for any of our other dependencies?

More concerning, would this change also break our customer's apps if they don't already have this dependency?

Reply:
Yea, I am not sure why this dependency is different. I'll take another look.